win32/Make.rules.mak

```diff
@@ -126,6 +126,10 @@ OPENPACE_DIR = C:\openpace
 !ENDIF
 OPENPACE_INCL_DIR = /I$(OPENPACE_DIR)\src
 OPENPACE_LIB = $(OPENPACE_DIR)\src\libeac.lib
+!IF "$(OPENSSL_DEF)" == "/DENABLE_OPENSSL"
```

Comment:
This should check for `OPENPACE_DEF`, I think

Reply:
This is the case when `npa-tool` needs OpenSSL and OpenPACE, as in the `Makefile.am`

https://github.com/OpenSC/OpenSC/blob/ee0c1fbacc4dc75ce52910564ef619db5503716a/src/tools/Makefile.am#L34-L40